gettsim/transfers/grundrente.py

```diff
@@ -0,0 +1,391 @@
+    does not lower the Grundrentenzuschlag.
+
+    The Deutsche Rentenversicherung uses the income of the year two to three years
+    ago to be able to use administrative data on this income for the calculation.
+
+    "Es ist davon auszugehen, dass dem Finanzamt regelmäßig zwei Jahre nach dem Ablauf
+    des Veranlagungszeitraums die Daten vorliegen, die von der Rentenversicherung
+    abgerufen werden können. "
+
+    Warning: Currently, earnings of dependend work and pensions are based on the
+    last year, and other income on the current year instead of the year
+    two years ago to avoid the need for several new input variables.
+
+    #ToDo: Freibeträge for income are currently not considered
+    #ToDO: as freibetraege_tu depends on pension income
+
+    Parameters
+    ----------
+    proxy_rente_vorj_excl_grundr_zuschlag_m
+        See :func:`proxy_rente_vorj_excl_grundr_zuschlag_m`.
+    bruttolohn_vorj_m
+        See :func:`bruttolohn_vorj_m`.
+    brutto_eink_1
+        See :func:`brutto_eink_1`.
+    brutto_eink_6
+        See :func:`brutto_eink_6`.
+    kapital_eink_minus_pauschbetr
+        See :func:`kapital_eink_minus_pauschbetr`.
+    Returns
+    -------
+    """
+
+    # Earnings from self-employed work and rent income (assumption that they were
+    # the same last year)
+    earnings_self_empl_rent = brutto_eink_1 + brutto_eink_6
+
+    out = (
+        proxy_rente_vorj_excl_grundr_zuschlag_m
+        + bruttolohn_vorj_m
+        + earnings_self_empl_rent / 12
+        + kapital_eink_minus_pauschbetr / 12
+    )
+
+    return out
+
+
+def proxy_rente_vorj_excl_grundr_zuschlag_m(
+    wohnort_ost: BoolSeries,
+    prv_rente_m: FloatSeries,
+    jahr_renteneintr: IntSeries,
+    geburtsjahr: IntSeries,
+    alter: IntSeries,
+    entgeltpunkte: FloatSeries,
+    zugangsfaktor: FloatSeries,
+    ges_renten_vers_params: dict,
+) -> FloatSeries:
+    """Estimated amount of public pensions of last year excluding Grundrentenzuschlag.
+
+    See params documentation :ref:`ges_renten_vers_params <ges_renten_vers_params>`.
+    wohnort_ost
+        See basic input variable :ref:`wohnort_ost <wohnort_ost>`.
+    prv_rente_m
+        See basic input variable :ref:`prv_rente_m <prv_rente_m>`.
+    jahr_renteneintr
+        See basic input variable :ref:`jahr_renteneintr <jahr_renteneintr>`.
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    alter
+        See basic input variable :ref:`alter <alter>`.
+    entgeltpunkte
+        See basic input variable :ref:`entgeltpunkte <entgeltpunkte>`.
+    zugangsfaktor
+        See :func:`zugangsfaktor`.
+    ges_renten_vers_params
+        See params documentation :ref:`ges_renten_vers_params <ges_renten_vers_params>`.
+
+    Returns
+    -------
+    """
+
+    # Calculate pension in the last year in case the subject was already retired
+    rentenwert_vorjahr = wohnort_ost.replace(
```

Comment:
We have an extra function for rentenwert. For consistency then also this as an extra function?